keps/sig-apps/4212-declarative-node-maintenance/kep.yaml

```diff
@@ -0,0 +1,49 @@
+title: Declarative Node Maintenance
+kep-number: 4212
+authors:
+  - "@atiratree"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-cli
+  - sig-apps
+  - sig-node
+  - sig-scheduling
+status: provisional
+creation-date: 2023-09-15
+reviewers:
+  - "@sftim"
+  - "@fabiand"
+  - "@mmerkes"
+  - "@rptaylor"
+  - "@pwschuurman"
+  - "@soltysh"
+approvers:
```

Comment:
You can add me as the approver for this KEP, but I think it might be too late for 1.30.

Reply:
thanks for signing up, I have added you there. Yeah 1.30 is probably a bit of stretch